content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -296,27 +298,23 @@ manager, and scheduler run as [DaemonSet pods](/docs/concepts/workloads/controll
 configured via the Kubernetes API instead of [static pods](/docs/tasks/administer-cluster/static-pod/)
 configured in the kubelet via static files.
 
-To create a self-hosted cluster, pass the flag `--feature-gates=SelfHosting=true` to `kubeadm init`.
-
-{{< caution >}}
-`SelfHosting` is an alpha feature. It is deprecated in 1.12
-and will be removed in 1.13.
-{{< /caution >}}
+To create a self-hosted cluster see the `kubeadm alpha selfhosting` command.
```

Comment:
Any reason to keep the docs for self hosting? iirc we only kept the alpha phase for existing users and without any real support, I don't think we want to onboard any new users to self-hosting.

Reply:
i'm OK with removing them, but i'm not going to do so without the consent of the group.
so possibly we can do that later in 1.15 and have meeting agenda items for it.

Reply:
+1

Reply:
I'm ok with moving it into something like a FAQ, but not nuking b/c every cycle we get asked about this.